google/cloud/apigee/registry/v1/registry_internal.proto

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.cloud.apigee.registry.v1;
+
+option java_package = "com.google.cloud.apigee.registry.v1";
+option java_multiple_files = true;
+option java_outer_classname = "RegistryInternalProto";
+option go_package = "github.com/apigee/registry/rpc;rpc";
+
+// OrderedPair represents a name-value pair in which both are strings.
+// (-- api-linter: core::0123::resource-annotation=disabled
+//     aip.dev/not-precedent: This message is not currently used in an API. --)
+message OrderedPair {
```

Comment:
Is this called OrderedPair because it's used in the OrderedMap? Having this message nested inside the OrderedMap message and named Pair or Entry seems more appropriate if it exists specifically for that use case.

Reply:
Yes, and I agree - will change.

Reply:
Deleted in the latest commit.